src/Containers/Panel/Panel.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback, useEffect } from 'react';
+import styled from 'styled-components';
+import { IPanelProps, OperationState } from '../FileUploadV2/FileUploadV2';
+import { Card as C } from '../Card/Card';
+import { Button as B } from '../../Inputs/Button/Button';
+
+export const Panel: React.FC<IPanelProps> = ({
+    onCancelUploading = () => null,
+    panel = null,
+    ...props
+}) => {
+    const getMessage = useCallback((): string => {
+        if (panel) {
+            switch (panel.operationState) {
+            case OperationState.isFailure:
+                return 'something went wrong';
+            case OperationState.isSuccess:
+                return 'completed';    
+            case OperationState.isLoading:
+                return `loading ${panel.name} ...`;
+            default:
+                return '';
+            }
+        }
+        return '';
+    }, [panel]);
+
+    useEffect(
+        () => () => {
+            if (panel) {
+                panel.worker?.terminate();
+            }
+        },
+        [],
+    );
+
+    return (
+        <Card {...props}>
+            <div>{getMessage()}</div>
+            {panel?.operationState === OperationState.isLoading && (
```

Comment:
replace with isLoading prop

Reply:
isLoading = True
is Loading = False

isSuccess = null
isError = null

isSuccess= 'text' | isSuccess=''completed''